nistats/contrasts.py

```diff
@@ -98,9 +99,10 @@ def _fixed_effect_contrast(labels, results, con_vals, contrast_type=None):
             contrast = contrast_
         else:
             contrast = contrast + contrast_
+        n_contrasts += 1
     if contrast is None:
         raise ValueError('all contrasts provided were null contrasts')
-    return contrast
+    return contrast * (1. / n_contrasts)
```

Comment:
Why not just return `contrast / len(con_vlas)`  ? (and get rid of `n_contrasts` alltogether)

Reply:
because (if I understand correctly) if some contrasts contain only zeros (users
can provide different contrasts for each run), the corresponding runs are
skipped:

https://github.com/nistats/nistats/blob/b69e127020df05aa8ba540f9a8d799e1dad3d602/nistats/contrasts.py#L95

so the number of contrasts that are summed can be smaller than e.g.
`len(con_vals)`.